earthpy/tests/test_spatial.py

```diff
@@ -108,6 +109,63 @@ def test_extent_to_json():
         es.extent_to_json([0, 1, 1, 0])
 
 
+def test_normalized_diff_shapes():
+    """If provided with arrays of different shapes,
+    normalized_diff returns ValueError."""
+
+    # Test data
+    b1_2d = np.array([[6, 7, 8, 9, 10], [16, 17, 18, 19, 20]])
```

Comment:
i think these could be fixtures
```
@pytest.fixture
def b1_b2_arrs():
    b1 = np.array([[6, 7, 8, 9, 10], [16, 17, 18, 19, 20]])
    b2 = np.array([[1, 2, 3, 4, 5], [14, 12, 13, 14, 17]])
    return b1, b2
```

function would then look like
```
def test_normalized_diff_shapes(b1_b2_arrs):
     """Test that two arrays with different shapes returns a ValueError."""

     # Test data
     b1, b2 = b1_b2_arrs
     b2 = b2[0] # or something like that to just select one part of the arr
 
     # Check ValueError
     with pytest.raises(ValueError, match=r"Both arrays should be of the same dimensions"):
         es.normalized_diff(b1 = b1, b2 = b2)
```

Reply:
I agree - using a fixture for these arrays will keep these tests nice and DRY :desert:

Reply:
how are you so good with emojis @mbjoseph  i don't even know how to find 🏜 !!

Reply:
Fixtures for these arrays have been implemented in testing of normalized_diff.